startup_scripts/200_vlan_groups.py

```diff
@@ -18,9 +18,17 @@
         if assoc in params:
             model, field = details
             query = {field: params.pop(assoc)}
-
-            params[assoc] = model.objects.get(**query)
-
+            # Get model from Contenttype
+            scope_type = params.pop("scope_type", None)
+            if not scope_type:
+                print("scope_type is missing from VLAN Group")
+                continue
+            app_label, model = str(scope_type).split(".")
+            ct = ContentType.objects.get(app_label=app_label, model=model)
+            if not ct:
+                print(f"ContentType for app_label = '{app_label}' and model = '{model}' not found")
```

Comment:
Here it would be good to know that this relates to a VLAN Group as well, and better if we would know which VLAN Group.